### ISSUE TITLE
Error in (function (experiments = ExperimentList(), colData = S4Vectors::DataFrame()

### ISSUE THREAD

**Issue by lwaldron** [waldronlab/curatedTCGAData#4]
Maybe jumping the gun here, but:

```
> curatedTCGAData("ACC", dry.run=FALSE, assays=c("CNASNP", "Mutation", "RPPAArray"))
snapshotDate(): 2017-10-12
snapshotDate(): 2017-10-12
see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation
loading from cache ‘/Users/lwaldron//.ExperimentHub/558’
snapshotDate(): 2017-10-12
see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation
downloading from ‘https://experimenthub.bioconductor.org/fetch/566’
retrieving 1 resource
  |===========================================================================================================| 100%

snapshotDate(): 2017-10-12
see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation
downloading from ‘https://experimenthub.bioconductor.org/fetch/568’
retrieving 1 resource
  |===========================================================================================================| 100%

Loading required package: RaggedExperiment
in method for ‘coerce’ with signature ‘"RangedRaggedAssay","RaggedExperiment"’: no definition for class “RaggedExperiment”
snapshotDate(): 2017-10-12
see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation
downloading from ‘https://experimenthub.bioconductor.org/fetch/560’
retrieving 1 resource
  |===========================================================================================================| 100%

snapshotDate(): 2017-10-12
see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation
downloading from ‘https://experimenthub.bioconductor.org/fetch/563’
retrieving 1 resource
  |===========================================================================================================| 100%

snapshotDate(): 2017-10-12
see ?curatedTCGAData and browseVignettes('curatedTCGAData') for documentation
downloading from ‘https://experimenthub.bioconductor.org/fetch/569’
retrieving 1 resource
  |===========================================================================================================| 100%

Error in (function (experiments = ExperimentList(), colData = S4Vectors::DataFrame(),  : 
  unused arguments (`ACC_colData-20160128` = <S4 object of class "DataFrame">, `ACC_metadata-20160128` = list(buildDate = 1506452819.92055, cancerCode = "ACC", runDate = "20160128", analyzeDate = "20160128", session_info = list(platform = list(version = "R Under development (unstable) (2017-04-12 r72509)", system = "x86_64, linux-gnu", ui = "X11", language = "(EN)", collate = "en_US.UTF-8", tz = "posixrules", date = "2017-09-26"), packages = list(package = c("AnnotationDbi", "AnnotationForge", "AnnotationHub", 
"AnnotationHubData", "assertthat", "base", "bindr", "bindrcpp", "Biobase", "BiocGenerics", "BiocInstaller", "BiocParallel", "biocViews", "biomaRt", "Biostrings", "bit", "bit64", "bitops", "blob", "cellranger", "compiler", "data.table", "datasets", "DBI", "DelayedArray", "devtools", "digest", "dplyr", "futile.logger", "futile.options", "GenomeInfoDb", "GenomeInfoDbData", "GenomicAlignments", "GenomicFeatures", "GenomicRanges", "GEOquery", "glue", "graph", "graphics", "grD
In addition: Warning messages:
1: database may not be current
  database: ‘/Users/lwaldron//.ExperimentHub/experimenthub.sqlite3’
  reason: Timeout was reached: SSLRead() return error -9806 
```

**Comment by LiNk-NY** [comment]
This was fixed here: 617311d6f5ecd78b03ab3d39b66fd03a4bd92470. 
Thanks,
Marcel

**Comment by LiNk-NY** [comment]
FWIW, The package tests will fail until Lori can update the datasets on ExperimentHub.
